src/components/hub/chatForm/ChatFormInputs.js

```diff
@@ -23,7 +23,9 @@ import { useEffect, useMemo, useRef, useState } from "react";
 import "@styles/hub/chatForm/ChatFormInputs.css";
 
 export default function ChatFormInputs({
+  opponentId,
```

Comment:
Can we get `opponentId` from existing selectors we use already in this component?
and also we can probably get `isConversationEncrypted ` as well, 
hence no need to introduce new props 

```
const participants = useSelector(selectParticipantsEntities);
  const selectedConversation = useSelector(getConverastionById);
```